pimcore/lib/Pimcore/Bundle/CoreBundle/Command/ThumbnailsImageCommand.php

```diff
@@ -138,7 +138,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
                         )
                     );
                     $progress->setMessage(
-                        'generated thumbnail: '.$image->getThumbnail($thumbnail)->getFilesystemPath()
+                        'generated thumbnail: ' . str_replace(PIMCORE_PROJECT_ROOT.'/', '', $image->getThumbnail($thumbnail)->getFilesystemPath())
```

Comment:
isn't that the same as getFullPath?

Reply:
You mean `getPath()`? No, it isn't `getPath()` is escaped (for use in the frontend).

Reply:
oh yeah, I see.